### PR TITLE
Process proctored exam grade report from edX

### LINKED ISSUE #4668: Process exam grades csv from edx
Add a management command for reading and creating (already adjusted) exam grades from a csv file provided by the course team.

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
Fix #4668

#### What's this PR do?
Add a management command for processing  proctored exam grade report form edX

#### How should this be manually tested?
In django shell set up an ExamRun that is currently schedulable. Make sure your user paid for the course.
Run the command `./manage.py import_edx_exam_grades 14.100`

### CODE REVIEW

File: exams/management/commands/import_edx_exam_grades.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    def add_arguments(self, parser):
+        parser.add_argument('csvfile', type=argparse.FileType('r'), help='')
+
+        parser.add_argument('course_code', type=argparse.FileType('r'), help='Example: 14.100')
+
+    def handle(self, *args, **kwargs):  # pylint: disable=unused-argument,too-many-locals
+
+        csvfile = kwargs.get('csvfile')
+        course_number = kwargs.get('course_code')
+        reader = csv.DictReader(csvfile.read().splitlines())
```
Comment (noisecapella) [inline exams/management/commands/import_edx_exam_grades.py:29]: It doesn't look like you need the `read().splitlines()` part here. `DictReader` can read from file handles directly

File: exams/management/commands/import_edx_exam_grades.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+
+class Command(BaseCommand):
+    """Parses a csv with exam grades creating or updating ProctoredExamGrade"""
+    help = "Parses a csv with exam grades and creates ProctoredExamGrade"
+
+    def add_arguments(self, parser):
+        parser.add_argument('csvfile', type=argparse.FileType('r'), help='')
+
+        parser.add_argument('course_code', type=argparse.FileType('r'), help='Example: 14.100')
```
Comment (noisecapella) [inline exams/management/commands/import_edx_exam_grades.py:23]: Is this supposed to be a `FileType`?

File: exams/management/commands/import_edx_exam_grades.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        except Course.DoesNotExist:
+            raise CommandError(
+                'Could not find a course with number "{}"'.format(course_number)
+            )
+        except Course.MultipleObjectsReturned:
+            raise CommandError(
+                'There are multiple courses with given number "{}"'.format(course_number)
+            )
+        # should be only one current exam run
+        exam_run = ExamRun.get_currently_schedulable(course).first()
```
Comment (noisecapella) [inline exams/management/commands/import_edx_exam_grades.py:42]: Can we use `except ExamRun.DoesNotExist` here, and also `ExamRun.MultipleObjectsReturned` to detect if there are more than one matching `ExamRun`?

**Review by noisecapella — CHANGES_REQUESTED** [review]

**Comment by annagav** [comment]
@noisecapella I addressed all your comments. Thank you!

**Review by noisecapella — APPROVED** [review]

**Pull request merged**